Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -35,4 +28,56 @@ function GetVssConnection
     return $vssConnection
 }
 
+#
+# The issues, as reported by SonarQube, need to be transformed to a simpler structure that the PostComments module can consume
+# 
+function GetCommentsFromIssues
+{
+    param ([Array]$issues)
+    
+    Write-Verbose "Transforming SonarQube analysis issues to PR comments"
+    
+    $comments = New-Object System.Collections.ArrayList
+
+    foreach ($issue in $issues)
+    {
+        $priority = GetCommentPriority $issue
+
+        Assert ( ![String]::IsNullOrWhiteSpace($issue.message) ) "Internal error: the SonarQube reported issues do not have a property named 'message'"
```

Comment:
Assert is a function I defined myself, there isn't one available in PS. It just throws.